cmake/developer_package/ncc_naming_style/ncc_naming_style.cmake

```diff
@@ -107,8 +107,10 @@ function(ov_ncc_naming_style)
 
     list(APPEND NCC_STYLE_ADDITIONAL_INCLUDE_DIRECTORIES "${NCC_STYLE_SOURCE_DIRECTORY}")
 
+# without it sources with same name from different directories will map to same .ncc_style target
+    file(RELATIVE_PATH source_dir_rel ${CMAKE_SOURCE_DIR} ${NCC_STYLE_SOURCE_DIRECTORY})
     foreach(source IN LISTS sources)
-        set(output_file "${ncc_style_bin_dir}/${source}.ncc_style")
+        set(output_file "${ncc_style_bin_dir}/${source_dir_rel}/${source}.ncc_style")
```

Comment:
could you investigate if naming conv bot failed because of your changes? In any case better to move it to the separate PR

Reply:
my change uncovered ncc check failure. i can move it to separate pr, but i can't build openvino locally without this fix